lib/common/utils/misc.dart

```diff
@@ -35,4 +36,12 @@ class Misc {
 
     return files;
   }
+
+  /// Returns (isLightMode, colorScheme)
```

Comment:
Lovely approach.

Reply:
Renamed the title. Public repos allow for PRs to be marked as drafts so no need for prefixing the `[WIP]`

Reply:
@root458,

Kindly update your local branch with `origin/main`. Just merged in some critical work on the feed and session details page. This should allow you to move forward much faster with the theming work. 

Key points to note:
- You are free to delete any code that needs deleting or that isn't useful
- Are you able to provide an estimate of when we can expect the final version for merging? Asking because there's another feature #24 that's in the works and someone has been assigned to it, but they can't make meaningful progress as merging will become trickly.

Reply:
Thank you @MillerAdulu 

Let's work with Monday night. I will also look at the codebase and find areas that can do with refactors.